### ISSUE TITLE
Update the installation requirements

### ISSUE THREAD

**Issue by Xinghui-Li** [NVIDIA/MinkowskiEngine#304]
Would you mind updating the installation requirement since v0.5 only support pytorch>=1.7 rather than >=1.5.

**Comment by wzm2256** [comment]
Is it necessary? I spent a few days installing this package, and it always failed with all types of errors. I'm not sure whether it is related to the version of pytorch.

**Comment by Xinghui-Li** [comment]
I exactly follow the installation guidance and it works well. I am pretty sure the current version only support pytorch>=1.7 since it uses nn.SiLU which is introduced in pytorch 1.7. It also has many other issues, like there is no clear way to move sparse tensor from gpu back to cpu, and you cannot convert the tensor to pytorch tensor if the tensor is on gpu.

**Comment by wzm2256** [comment]
@Xinghui-Li Did you install this package use anaconda? Could you kindly share your conda list? Since the instruction does not mention the version of everything, things still go wrong even if I follow the instruction.

**Comment by Xinghui-Li** [comment]
Yes, I used anaconda. My OS is Ubuntu 18.04, cuda is 10.2. My conda list is awfully long since I also installed other packages.

# Name                    Version                   Build  Channel                                                                                                                                                                                                                 
_libgcc_mutex             0.1                        main                                                                                                                                                                                                                          
blas                      1.0                         mkl
ca-certificates           2020.10.14                    0    anaconda
certifi                   2020.6.20                py38_0    anaconda
cudatoolkit               10.1.243             h6bb024c_0
cycler                    0.10.0                     py_2    conda-forge
dataclasses               0.8                      pypi_0    pypi
dbus                      1.13.18              hb2f20db_0
expat                     2.2.10               he6710b0_2
fontconfig                2.13.1            he4413a7_1000    conda-forge
freetype                  2.10.4               h7ca028e_0    conda-forge
future                    0.18.2                   pypi_0    pypi
glib                      2.66.1               h92f7085_0
gst-plugins-base          1.14.0               hbbd80ab_1
gstreamer                 1.14.0               h28cd5cc_2
icu                       58.2              hf484d3e_1000    conda-forge
intel-openmp              2020.2                      254
jpeg                      9d                   h36c2ea0_0    conda-forge
kiwisolver                1.3.1            py38h82cb98a_0    conda-forge
ld_impl_linux-64          2.33.1               h53a641e_7
libedit                   3.1.20191231         h14c3975_1
libffi                    3.3                  he6710b0_2
libgcc-ng                 9.1.0                hdf63c60_0
libgfortran-ng            8.2.0                hdf63c60_1    anaconda
libopenblas               0.3.2                h9ac9557_1    anaconda
libpng                    1.6.37               h21135ba_2    conda-forge
libstdcxx-ng              9.1.0                hdf63c60_0
libtiff                   4.0.10            hc3755c2_1005    conda-forge
libuuid                   2.32.1            h14c3975_1000    conda-forge
libuv                     1.40.0               h7b6447c_0
libxcb                    1.13              h14c3975_1002    conda-forge
libxml2                   2.9.10               hb55368b_3
lz4-c                     1.9.2                he1b5a44_3    conda-forge
matplotlib                3.3.3            py38h578d9bd_0    conda-forge
matplotlib-base           3.3.3            py38h5c7f4ab_0    conda-forge
minkowskiengine           0.5.0                    pypi_0    pypi
mkl                       2020.2                      256
mkl-service               2.3.0            py38he904b0f_0
mkl_fft                   1.2.0            py38h23d657b_0
mkl_random                1.1.1            py38h0573a6f_0
ncurses                   6.2                  he6710b0_1
ninja                     1.10.2           py38hff7bd54_0
numpy                     1.19.2           py38h54aff64_0
numpy-base                1.19.2           py38hfa32c7d_0
olefile                   0.46               pyh9f0ad1d_1    conda-forge
openblas-devel            0.3.2                         0    anaconda
openssl                   1.1.1h               h7b6447c_0    anaconda
pcre                      8.44                 he1b5a44_0    conda-forge
pillow                    6.2.1            py38h6b7be26_0    conda-forge
pip                       20.3.3           py38h06a4308_0
pthread-stubs             0.4               h36c2ea0_1001    conda-forge
pyparsing                 2.4.7              pyh9f0ad1d_0    conda-forge
pyqt                      5.9.2            py38h05f1152_4
python                    3.8.5                h7579374_1
python-dateutil           2.8.1                      py_0    conda-forge
python_abi                3.8                      1_cp38    conda-forge
pytorch                   1.7.0           py3.8_cuda10.1.243_cudnn7.6.3_0    pytorch
qt                        5.9.7                h5867ecd_1
readline                  8.0                  h7b6447c_0
setuptools                51.3.3           py38h06a4308_4
sip                       4.19.13          py38he6710b0_0
six                       1.15.0           py38h06a4308_0
sqlite                    3.33.0               h62c20be_0
tk                        8.6.10               hbc83047_0
torchvision               0.8.1                py38_cu101    pytorch
tornado                   6.1              py38h25fe258_0    conda-forge
typing_extensions         3.7.4.3                    py_0
wheel                     0.36.2             pyhd3eb1b0_0
xorg-libxau               1.0.9                h14c3975_0    conda-forge
xorg-libxdmcp             1.1.3                h516909a_0    conda-forge
xz                        5.2.5                h7b6447c_0
zlib                      1.2.11               h7b6447c_3
zstd                      1.4.5                h9ceee32_0

**Comment by Xinghui-Li** [comment]
Yes, I used anaconda. My OS is Ubuntu 18.04, cuda is 10.2. My conda list is awfully long since I also installed other packages.

_libgcc_mutex 0.1 main
blas 1.0 mkl
ca-certificates 2020.10.14 0 anaconda
certifi 2020.6.20 py38_0 anaconda
cudatoolkit 10.1.243 h6bb024c_0
cycler 0.10.0 py_2 conda-forge
dataclasses 0.8 pypi_0 pypi
dbus 1.13.18 hb2f20db_0
expat 2.2.10 he6710b0_2
fontconfig 2.13.1 he4413a7_1000 conda-forge
freetype 2.10.4 h7ca028e_0 conda-forge
future 0.18.2 pypi_0 pypi
glib 2.66.1 h92f7085_0
gst-plugins-base 1.14.0 hbbd80ab_1
gstreamer 1.14.0 h28cd5cc_2
icu 58.2 hf484d3e_1000 conda-forge
intel-openmp 2020.2 254
jpeg 9d h36c2ea0_0 conda-forge
kiwisolver 1.3.1 py38h82cb98a_0 conda-forge
ld_impl_linux-64 2.33.1 h53a641e_7
libedit 3.1.20191231 h14c3975_1
libffi 3.3 he6710b0_2
libgcc-ng 9.1.0 hdf63c60_0
libgfortran-ng 8.2.0 hdf63c60_1 anaconda
libopenblas 0.3.2 h9ac9557_1 anaconda
libpng 1.6.37 h21135ba_2 conda-forge
libstdcxx-ng 9.1.0 hdf63c60_0
libtiff 4.0.10 hc3755c2_1005 conda-forge
libuuid 2.32.1 h14c3975_1000 conda-forge
libuv 1.40.0 h7b6447c_0
libxcb 1.13 h14c3975_1002 conda-forge
libxml2 2.9.10 hb55368b_3
lz4-c 1.9.2 he1b5a44_3 conda-forge
matplotlib 3.3.3 py38h578d9bd_0 conda-forge
matplotlib-base 3.3.3 py38h5c7f4ab_0 conda-forge
minkowskiengine 0.5.0 pypi_0 pypi
mkl 2020.2 256
mkl-service 2.3.0 py38he904b0f_0
mkl_fft 1.2.0 py38h23d657b_0
mkl_random 1.1.1 py38h0573a6f_0
ncurses 6.2 he6710b0_1
ninja 1.10.2 py38hff7bd54_0
numpy 1.19.2 py38h54aff64_0
numpy-base 1.19.2 py38hfa32c7d_0
olefile 0.46 pyh9f0ad1d_1 conda-forge
openblas-devel 0.3.2 0 anaconda
openssl 1.1.1h h7b6447c_0 anaconda
pcre 8.44 he1b5a44_0 conda-forge
pillow 6.2.1 py38h6b7be26_0 conda-forge
pip 20.3.3 py38h06a4308_0
pthread-stubs 0.4 h36c2ea0_1001 conda-forge
pyparsing 2.4.7 pyh9f0ad1d_0 conda-forge
pyqt 5.9.2 py38h05f1152_4
python 3.8.5 h7579374_1
python-dateutil 2.8.1 py_0 conda-forge
python_abi 3.8 1_cp38 conda-forge
pytorch 1.7.0 py3.8_cuda10.1.243_cudnn7.6.3_0 pytorch
qt 5.9.7 h5867ecd_1
readline 8.0 h7b6447c_0
setuptools 51.3.3 py38h06a4308_4
sip 4.19.13 py38he6710b0_0
six 1.15.0 py38h06a4308_0
sqlite 3.33.0 h62c20be_0
tk 8.6.10 hbc83047_0
torchvision 0.8.1 py38_cu101 pytorch
tornado 6.1 py38h25fe258_0 conda-forge
typing_extensions 3.7.4.3 py_0
wheel 0.36.2 pyhd3eb1b0_0
xorg-libxau 1.0.9 h14c3975_0 conda-forge
xorg-libxdmcp 1.1.3 h516909a_0 conda-forge
xz 5.2.5 h7b6447c_0
zlib 1.2.11 h7b6447c_3
zstd 1.4.5 h9ceee32_0
